lib/avrolution/rake/add_compatibility_break_task.rb

```diff
@@ -6,8 +6,8 @@ module Avrolution
   module Rake
     class AddCompatibilityBreakTask < BaseTask
 
-      def initialize(*)
-        super
+      def initialize(**options)
```

Comment:
Did you need the `options` variable here? Would this work:
```ruby
def initialize(**)
  super
  # ...
end
```

Reply:
IntelliJ thinks `**` is an incorrect extra argument, but the code seems to work fine.